requirements.txt

```diff
@@ -1,13 +1,14 @@
-fbs~=1.2
-PyInstaller~=3.4
-pyqt5==5.9.2
+#fbs~=1.2
+fbs @ git+https://github.com/t0mpr1c3/fbs@py38
```

Comment:
We should choose a sensible tag name in the ayab fbs fork repo to identify the version and use it here to specify the version

Reply:
Yes

Reply:
Let's use the tag 'ayab-py38'